src/app/portlets/dot-edit-page/components/dot-edit-page-info/dot-edit-page-info.component.spec.ts

```diff
@@ -76,7 +76,7 @@ describe('DotEditPageInfoComponent', () => {
 
         it('should have api link', () => {
             const apiLink: DebugElement = de.query(By.css('dot-api-link'));
-            expect(apiLink.componentInstance.href).toBe(
+            expect(apiLink.componentInstance.link).toBe(
```

Comment:
any specific reason, why it is changed to `link`?

Reply:
Because of the setter, Im getting the internal property of the class.